job_creator.php

```diff
@@ -379,6 +387,13 @@ private function createPhpunitJobs(
                     'phpunit' => true,
                     'phpunit_suite' => $suite,
                 ]);
+                if ($this->getBranchVersion() >= '5.2') {
```

Comment:
I tested this on personal fork, but I had to change this line to `if ($this->getBranchVersion() === '5.1')`. 
All failed tests are not related to PHP 8.3 support.
https://github.com/sabina-talipova/recipe-kitchen-sink/actions/runs/7177998490/job/19545421461